github/resource_github_project_card_test.go

```diff
@@ -0,0 +1,129 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/google/go-github/v31/github"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccGithubProjectCard_basic(t *testing.T) {
+	var card github.ProjectCard
+
+	rn := "github_project_card.card"
+	note := "## Unaccepted :point_down:"
+	updatedNote := "## Accepted :point_up:"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccGithubProjectCardDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGithubProjectCardConfig(note),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGithubProjectCardExists(rn, &card),
+					testAccCheckGithubProjectCardAttributes(&card, &testAccGithubProjectCardExpectedAttributes{
+						Note: note,
+					}),
+				),
+			},
+			{
+				Config: testAccGithubProjectCardConfig(updatedNote),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGithubProjectCardExists(rn, &card),
+					testAccCheckGithubProjectCardAttributes(&card, &testAccGithubProjectCardExpectedAttributes{
+						Note: updatedNote,
+					}),
+				),
+			},
```

Comment:
we'll need an ImportStep here to verify but I believe the resource's Importer func will need an update as well